packages/create-svelte/templates/skeletonlib/package.template.json

```diff
@@ -3,17 +3,32 @@
 	"version": "0.0.1",
 	"scripts": {
 		"dev": "vite dev",
-		"build": "svelte-kit sync && svelte-package",
-		"prepublishOnly": "echo 'Did you mean to publish `./package/`, instead of `./`?' && exit 1"
+		"build": "vite build && npm run package",
+		"preview": "vite preview",
+		"package": "svelte-kit sync && svelte-package && publint",
+		"prepublishOnly": "npm run package && publint"
+	},
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"svelte": "./dist/index.js",
+			"default": "./dist/index.js"
+		}
+	},
+	"peerDependencies": {
+		"svelte": "^3.54.0"
 	},
 	"devDependencies": {
 		"@sveltejs/adapter-auto": "^2.0.0",
 		"@sveltejs/kit": "^1.5.0",
 		"@sveltejs/package": "^1.0.0",
+		"publint": "^0.1.9",
 		"svelte": "^3.54.0",
 		"tslib": "^2.4.1",
 		"typescript": "^4.9.3",
 		"vite": "^4.0.0"
 	},
+	"svelte": "./dist/index.js",
+	"types": "./dist/index.d.ts",
```

Comment:
this is harmless but is there any tooling that recognises `types` but not `exports[...].types`? if not then maybe we can remove it?

```suggestion
```

Reply:
That "any tooling" is current typescript's default. It won't look at exports unless you opt into a different module resolution behavior. So we absolutely need to keep it